requirements.txt

```diff
@@ -5,4 +5,5 @@
 apicapi
 oslo.concurrency>=1.8.0,<1.9.0
 
--e git+https://github.com/noironetworks/python-opflex-agent.git@stable/juno#egg=opflexagent
+-e git://github.com/openstack/neutron.git@stable/juno#egg=neutron
```

Comment:
Don't we assume that neutron is installed in the system when loading the driver? Why are we moving this in the requirements?